src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -145,6 +145,16 @@ class CompanyStep extends React.Component {
     validate() {
         const errors = {};
 
+        if (this.state.manualAddress) {
```

Comment:
I think we can probably come up with a more descriptive name than manualAddress, perhaps something like `isManualAddress` since this is a boolean, not the actual address.